src/app/loginComponents/accounts/internal/accounts.component.html

```diff
@@ -25,16 +25,25 @@ <h3>Dockstore Profile</h3>
       <mat-list>
         <mat-list-item>
           <h1 matLine>
-            <strong>Username:</strong>
+            <mat-icon>account_circle</mat-icon> <strong class="align-middle">Username:</strong>
           </h1>
           <p matLine id="account-username">{{user?.username}}</p>
         </mat-list-item>
-        <mat-list-item>
-          <h1 matLine>
-            <strong>Administrator:</strong>
+        <mat-list-item *ngIf="user?.isAdmin || user?.curator">
+          <h1 matLine *ngIf="user?.isAdmin" id="account-is-admin">
+            <mat-icon>lock</mat-icon> <strong class="align-middle">Administrator</strong>
+          </h1>
+          <h1 matLine *ngIf="user?.curator && !user.isAdmin" id="account-is-curator">
+            <mat-icon >edit</mat-icon><strong class="align-middle">Curator</strong>
           </h1>
-          <p matLine id="account-is-admin">{{ user?.isAdmin ? 'Yes' : 'No' }}</p>
+          <!--<p matLine id="account-is-admin">{{ user?.isAdmin ? 'Yes' : 'No' }}</p>-->
         </mat-list-item>
+        <!--<mat-list-item>-->
+          <!--<h1 matLine>-->
+            <!--<mat-icon >edit</mat-icon><strong class="align-middle">Curator:</strong>-->
+          <!--</h1>-->
+          <!--&lt;!&ndash;<p matLine id="account-is-curator">{{ user?.curator ? 'Yes' : 'No' }}</p>&ndash;&gt;-->
+        <!--</mat-list-item>-->
```

Comment:
I would just get rid of the commented out lines unless you expect to restore them. If we really need to go back to them, they are in version control.